transactions/models.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from user.models import VirtualWalletUser
+
+
+class Wallet(models.Model):
+    currency_types = (
+        ('UAH', 'UAH'),
```

Comment:
It's nice to have constants for keys of this set. It's common to use these constants in the codebase for further development.